web/04-runpecan.php

```diff
@@ -68,7 +68,14 @@
 if (!isset($_REQUEST['ensemble'])) {
   die("Need an ensemble value.");
 }
-$ensemble=$_REQUEST['ensemble'];
+$ensemble = $_REQUEST['ensemble'];
+
+$sensitivity = array();
+if (!isset($_REQUEST['sensitivity'])) {
+  die("Need a sensitivity value-comma seperated.");
```

Comment:
Not clear from message what type of value is expected required here... are these quantiles in a comma separated list? You could say something like '1, 5, 33, 66, 95, 99 is the default'